modules/ROOT/pages/managing-your-account.adoc

```diff
@@ -107,29 +108,30 @@ Your username must be at least three characters and contain only the following:
 [NOTE]
 You cannot create a new account within an existing organization without an email invitation.
 
-== Log in to Anypoint Platform
+== Sign in to Anypoint Platform
 
-. Navigate to the Anypoint Platform login screen.
+. Navigate to the Anypoint Platform signin screen.
 . Enter your username and password.
 . Click *Sign in*.
 
 [NOTE]
-If you try to log in and Anypoint Platform rejects your valid credentials, your organization might use an xref:access-management::external-identity.adoc[external identity provider] to initiate the sign-in process. Either navigate to your external identity provider's interface to sign in, or xref:access-management::managing-your-account.adoc#use-CustomDomainLogin[enter your organization's domain name] before reentering your credentials.
+If you try to sign in and Anypoint Platform rejects your valid credentials, your organization might use an xref:access-management::external-identity.adoc[external identity provider] to initiate the sign-in process. Either navigate to your external identity provider's interface to sign in, or xref:access-management::managing-your-account.adoc#use-CustomDomainLogin[enter your organization's domain name] before reentering your credentials.
```

Comment:
I'm not sure we can use caps in link targets.

Reply:
Yeah, that's one that already existed and I didn't want to change existing links. But they do work. We have a ton like that in connectors (generated).